.vortex/installer/tests/Fixtures/install/provision_database_lagoon/docker-compose.yml

```
@@ -0,0 +1,95 @@
+ x-environment: &default-environment
+   TZ: ${TZ:-Australia/Melbourne}
+   # Local development URL.
+@@ -86,6 +87,10 @@
+     # Mount volumes from the ssh-agent running in Pygmy to inject host SSH key into container. See https://pygmy.readthedocs.io/en/master/ssh_agent/
+     volumes_from: ### Local overrides to mount host SSH keys. Automatically removed in CI.
+       - container:amazeeio-ssh-agent ### Local overrides to mount host SSH keys. Automatically removed in CI.
++    labels:
++      lagoon.type: cli-persistent # See https://docs.lagoon.sh/using-lagoon-advanced/service-types/
++      lagoon.persistent.name: &lagoon-nginx-name nginx-php # Mount the persistent storage of nginx-php pod into this container.
++      lagoon.persistent: &lagoon-persistent-files /app/web/sites/default/files/  # Location where the persistent storage should be mounted.
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)